doc/widgets.md

```diff
@@ -31,9 +40,12 @@ NineTileBox(
 
 ## SpriteButton
 
-`SpriteButton` is a simple widget that creates a button based on Flame sprites. This can be very useful when trying to create retro looking buttons. For example when creating that look can be more easily achievable by drawing the button in a drawing software, instead of making it directly in Flutter.
+`SpriteButton` is a simple widget that creates a button based on Flame sprites. This can be very
+useful when trying to create non-default looking buttons. For example when it is easier for you to
+achieve your wanted look by drawing the button in a graphics editor, instead of making it directly
+in Flutter.
 
-Remember that the Sprites must have already been loaded when passing it to the widget.
+Remember that the Sprites must already be loaded when passing them to the widget.
```

Comment:
I think this sentence is not needed anymore, we don't have the concept of an unloaded Sprite as the only way of getting a Sprite instance, is by passing an Image instance. So this sentence could be removed IMO

Reply:
Good point, fixed!